webapp/sources/rudder/rudder-rest/src/test/scala/com/normation/rudder/rest/TestRestPluginInfo.scala

```diff
@@ -140,7 +140,7 @@ class TestRestPluginInfo extends Specification {
     )
   )
 
-  val pluginApi = new PluginApi(null, null, pluginInfo.succeed)
+  val pluginApi = new PluginApi(null, null, null, pluginInfo.succeed)
```

Comment:
this looks like an anti-pattern. We should not have nulls in our tests. At least, it show that that part is not tested, but even without that, a constructor that works with 3/4 of its parameter set to null points to a bad splitting of responsibility. 

I don't have a good general answer, but at least for that one: 
- can you add a test for that API ? 
- can you add something that makes checkable that the effect was called ? (perhaps it's something like the date of last update returned in the api result, or a mock instance for `updateIndex` that check that it's called, or whatever that let us know it happened)